src/docker-in-docker/install.sh

```diff
@@ -244,73 +244,25 @@ echo "Finished installing docker / moby!"
-                echo "(*) Failed to install docker-compose via pipx. Trying via pip3..."
-
-                export PYTHONUSERBASE=/usr/local
-                pip3 install --disable-pip-version-check --no-cache-dir --user "Cython<3.0" pyyaml wheel docker-compose --no-build-isolation
-            fi
-
-            rm -rf /tmp/pip-tmp
-        else
-            compose_v1_version="1"
-            find_version_from_git_tags compose_v1_version "https://github.com/docker/compose" "tags/"
-            echo "(*) Installing docker-compose ${compose_v1_version}..."
-            curl -fsSL "https://github.com/docker/compose/releases/download/${compose_v1_version}/docker-compose-Linux-x86_64" -o /usr/local/bin/docker-compose
-            chmod +x /usr/local/bin/docker-compose
-        fi
+    if [ "${architecture}" == "amd64" ]; then
+        dockerComposeArch="x86_64"
+    elif [ "${architecture}" == "arm64" ]; then
+        dockerComposeArch="aarch64"
     fi
 
-    # Install docker-compose switch if not already installed - https://github.com/docker/compose-switch#manual-installation
-    current_v1_compose_path="$(which docker-compose)"
-    target_v1_compose_path="$(dirname "${current_v1_compose_path}")/docker-compose-v1"
-    if ! type compose-switch > /dev/null 2>&1; then
-        echo "(*) Installing compose-switch..."
-        compose_switch_version="latest"
-        find_version_from_git_tags compose_switch_version "https://github.com/docker/compose-switch"
-        curl -fsSL "https://github.com/docker/compose-switch/releases/download/v${compose_switch_version}/docker-compose-linux-${architecture}" -o /usr/local/bin/compose-switch
-        chmod +x /usr/local/bin/compose-switch
-        # TODO: Verify checksum once available: https://github.com/docker/compose-switch/issues/11
-
-        # Setup v1 CLI as alternative in addition to compose-switch (which maps to v2)
-        mv "${current_v1_compose_path}" "${target_v1_compose_path}"
-        update-alternatives --install /usr/local/bin/docker-compose docker-compose /usr/local/bin/compose-switch 99
-        update-alternatives --install /usr/local/bin/docker-compose docker-compose "${target_v1_compose_path}" 1
-    fi
-    if [ "${DOCKER_DASH_COMPOSE_VERSION}" = "v1" ]; then
-        update-alternatives --set docker-compose "${target_v1_compose_path}"
+    if [ "${DOCKER_DASH_COMPOSE_VERSION}" == "latest" ]; then
+        docker_compose_version="latest"
+        find_version_from_git_tags docker_compose_version "https://github.com/docker/compose" "refs/tags/v"
     else
-        update-alternatives --set docker-compose /usr/local/bin/compose-switch
+        docker_compose_version="${DOCKER_DASH_COMPOSE_VERSION}"
     fi
+
+    echo "(*) Installing docker-compose ${docker_compose_version}..."
+
+    curl --location https://github.com/docker/compose/releases/download/v${docker_compose_version}/docker-compose-linux-${dockerComposeArch} \
+        --output /usr/local/bin/docker-compose
```

Comment:
Hmmm, now that I think about it --> ^ would fail if anyone passes `v2` (including test scenarios) 

I wonder if we should deprecate passing `v2` as option? 🤔 One can pass semvers instead (eg. 2 or 2.19 or 2.19.0). This would allow us to sync with other Feature options which does not contain the `v` prefix.

As we are bumping major versions, we could sneak this change in  (if we think that's needed, else we can condition on the `"refs/tags/v"` bit! ) 😬 

// cc @Chuxel @craiglpeters @bamurtaugh for thoughts on this change.